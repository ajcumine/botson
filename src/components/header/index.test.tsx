import { shallow } from 'enzyme';
import React from 'react';
import Header from '.';

describe('Header component', () => {
  it('renders', () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
