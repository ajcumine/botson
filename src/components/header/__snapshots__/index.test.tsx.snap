// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders 1`] = `
<header>
  <img
    alt="logo"
    src="logo.svg"
  />
  <h1>
    Welcome to React
  </h1>
</header>
`;
